frontend/store/authorizations.ts

```diff
@@ -190,6 +237,19 @@ export const actions = {
             commit('setIsCollateralAuthorizationLoading', { collateralType, isLoading: false });
         }
     },
+    async authorizeDebtAuctionContract({ commit, dispatch, rootGetters }: ActionContext<State, State>) {
+        commit('setIsWalletAuthorizationLoading', true);
```

Comment:
I think it's wrong setter
```suggestion
        commit('setIsDebtAuctionAuthorizationLoading', true);
```
(also below)